demisto_sdk/commands/validate/validators/IN_validators/IN163_is_valid_feed_expiration_policy.py

```diff
@@ -0,0 +1,113 @@
+    )
+    rationale = (
+        "Malformed expiration policy can lead to errors or incomplete data."
+        "For more details, see https://xsoar.pan.dev/docs/integrations/feeds"
+    )
+    error_message = "The feed's expiration policy is not in the correct format."
+    related_field = "feedExpirationPolicy"
+    is_auto_fixable = True
+
+    def obtain_invalid_content_items(
+        self, content_items: Iterable[ContentTypes]
+    ) -> List[ValidationResult]:
+        invalid_content_items = []
+        for content_item in content_items:
+            incremental_feed_param = next(
+                (
+                    param
+                    for param in content_item.params
+                    if param.name == "feedIncremental"
+                    and param.hidden is True
+                    and param.type == 8
+                    and param.defaultvalue is True
+                ),
+                None,
+            )
+
+            expiration_policy = next(
+                (
+                    param
+                    for param in content_item.params
+                    if param.name == "feedExpirationPolicy"
+                ),
+                None,
+            )
+            if content_item.is_feed:
+                if not expiration_policy:
+                    invalid_content_items.append(
+                        ValidationResult(
+                            validator=self,
+                            message=self.error_message + MISSING_EXPIRATION_POLICY,
+                            content_object=content_item,
+                        )
+                    )
+
+                elif expiration_policy.display != "" or expiration_policy.type != 17:
+                    invalid_content_items.append(
+                        ValidationResult(
+                            validator=self,
+                            message=self.error_message + BAD_TYPE_OR_DISPLAY,
+                            content_object=content_item,
+                        )
+                    )
+
+                elif incremental_feed_param and is_sudden_death(expiration_policy):
+                    invalid_content_items.append(
+                        ValidationResult(
+                            validator=self,
+                            message=self.error_message
+                            + REDUNDANT_SUDDEN_DEATH_ERROR_MESSAGE,
+                            content_object=content_item,
+                        )
+                    )
+
+                elif not incremental_feed_param and not is_sudden_death(
+                    expiration_policy
+                ):
+                    invalid_content_items.append(
+                        ValidationResult(
+                            validator=self,
+                            message=self.error_message
+                            + MISSING_SUDDEN_DEATH_ERROR_MESSAGE,
+                            content_object=content_item,
```

Comment:
all of this is very long with a lot of copy paste, makes it hard to read the logic. Lets extract everything thats the same from the if statements and put at the end

so 
```
messag_in_ifs = ''
if ...
messag_in_ifs = 1
elif...
messag_in_ifs = 2....
```
...

```
contentitems.append(ValidationResult(
                            validator=self,
                            message=self.error_message
                            + **messag_in_ifs**,
                            content_object=content_item)
```

Reply:
done